### PR TITLE
Helm: Add support for OpenShift route for Nginx

### PR DESCRIPTION
**Pull request by radudd**
#### What this PR does
This PR introduces additional support for OpenShift by providing the possibility to create OpenShift Routes for the Nginx component.

#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: operations/helm/charts/mimir-distributed/values.yaml [hunk 1]
```diff
@@ -1620,6 +1620,19 @@ nginx:
...
         hosts:
           - nginx.loki.example.com
+  # Route configuration (for OpenShift only)
+  route:
+    # -- Specifies whether an OpenShift route for the nginx should be created
+    enabled: false
+    # -- Annotations for the nginx route
+    annotations: {}
+    # -- Hostname conifguration
+    host: nginx.loki.example.com
```
Comment (dimitarvdimitrov) [inline operations/helm/charts/mimir-distributed/values.yaml:1630]:
```suggestion
    host: nginx.mimir.example.com
```

File: operations/helm/charts/mimir-distributed/values.yaml [hunk 2]
```diff
@@ -1620,6 +1620,19 @@ nginx:
...
+    # -- TLS configuration for OpenShift Route
+    tls:
+      # More details about TLS configuration and termination types: https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#secured-routes
```
Comment (dimitarvdimitrov) [inline operations/helm/charts/mimir-distributed/values.yaml:1633]:
```suggestion
      # -- More details about TLS configuration and termination types: https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#secured-routes
```

File: operations/helm/charts/mimir-distributed/values.yaml [hunk 3]
```diff
@@ -1620,6 +1620,19 @@ nginx:
       - secretName: loki-nginx-tls
         hosts:
           - nginx.loki.example.com
+  # Route configuration (for OpenShift only)
```
Comment (dimitarvdimitrov) [inline operations/helm/charts/mimir-distributed/values.yaml:1623]:
```suggestion
  # -- Route configuration (for OpenShift only)
```

**Review by dimitarvdimitrov — APPROVED** [review]
LGTM, many thanks for the contribution! :)

File: operations/helm/charts/mimir-distributed/values.yaml [hunk 4]
```diff
@@ -1669,6 +1669,19 @@ nginx:
       - secretName: loki-nginx-tls
         hosts:
           - nginx.loki.example.com
+  # -- Route configuration (for OpenShift only)
+  route:
+    # -- Specifies whether an OpenShift route for the nginx should be created
+    enabled: false
+    # -- Annotations for the nginx route
+    annotations: {}
+    # -- Hostname conifguration
```
Comment (lamida) [inline operations/helm/charts/mimir-distributed/values.yaml:1678]:
```suggestion
    # -- Hostname configuration
```

**Comment by dimitarvdimitrov** [comment]
Have you managed to test this on an openshift cluster already @radudd ? I don't have access to one and asked @patrickabrennan to try with his.

**Comment by radudd** [comment]
> Have you managed to test this on an openshift cluster already @radudd ? I don't have access to one and asked @patrickabrennan to try with his.

Hey @dimitarvdimitrov. Yes, I tested it couple of times and it works.

**Pull request merged**